client/post-editor/editor-publish-button/index.jsx

```diff
@@ -87,8 +88,16 @@ export class EditorPublishButton extends Component {
 			case 'update':
 				return this.props.translate( 'Update' );
 			case 'schedule':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Schedule…',
+						{ comment: 'Button label on the editor sidebar - a confirmation step will follow' } );
+				}
 				return this.props.translate( 'Schedule' );
 			case 'publish':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Publish…',
```

Comment:
Hi! I've found a possible matching string that has already been translated **49** times:
`translate( 'Publish', { context: 'admin bar menu group label'} )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-sidebar-indication)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?